mlflow/tracking/fluent.py

```diff
@@ -5,30 +5,50 @@
 
 from __future__ import print_function
 
-import atexit
 import numbers
 import os
+
+import atexit
 import sys
 import time
 
 from mlflow.entities import Experiment, Run, SourceType
+from mlflow.tracking.client import MlflowClient
 from mlflow.utils import env
 from mlflow.utils.databricks_utils import is_in_databricks_notebook, get_notebook_id, \
     get_notebook_path, get_webapp_url
 from mlflow.utils.mlflow_tags import MLFLOW_DATABRICKS_WEBAPP_URL, \
     MLFLOW_DATABRICKS_NOTEBOOK_PATH, \
     MLFLOW_DATABRICKS_NOTEBOOK_ID
 from mlflow.utils.validation import _validate_run_id
-from mlflow.tracking.client import MlflowClient
-
 
 _EXPERIMENT_ID_ENV_VAR = "MLFLOW_EXPERIMENT_ID"
 _RUN_ID_ENV_VAR = "MLFLOW_RUN_ID"
 _active_run = None
+_active_experiment_id = None
+
+
+def set_experiment(experiment_name):
+    """
+    Set given experiment as active experiment. If experiment does not exist, create an experiment
+    with provided name.
+
+    :param experiment_name: Name of experiment to be activated.
+    """
+    client = MlflowClient()
+    experiment = client.get_experiment_by_name(experiment_name)
+    exp_id = experiment.experiment_id if experiment else None
+    if not exp_id:
+        print("INFO: '{}' does not exist. Creating a new experiment".format(experiment_name))
```

Comment:
Can we use `eprint` (`from mlflow.utils.logging_utils import eprint`) to avoid overriding the user's stdout?

Reply:
👍

Reply:
In general, it's nice of library code to use stderr (e.g., python `logging` does this). This is to help programs which use stdout for real output. In other words, stdout is typically used for "real output" and stderr is used for "meta output", such as logs. This use-case falls into the latter category.

Reply:
Agreed. We shud move to using `logging` library. Lemme create a new issue to track that so we can use it for both `MlflowClient` and server side.